CommonLibSF/include/RE/B/BSTEvent.h

```diff
@@ -36,11 +41,51 @@ namespace RE
 		// add
 		virtual BSEventNotifyControl ProcessEvent(const Event& a_event, BSTEventSource<Event>* a_source) = 0;  // 01
 	};
+	static_assert(sizeof(BSTEventSink<void*>) == 0x8);
 
 	template <class Event>
 	class BSTEventSource : public BSTEventDetail::SourceBase
 	{
 	public:
+		using event_type = Event;
```

Comment:
Why use an extra type alias here?

Reply:
[that's how Ryan set it up in CLIBF4](https://github.com/shad0wshayd3/CommonLibF4/blob/2e5b66108bc8cc125a32372f75865fd62c2d525e/CommonLibF4/include/RE/Bethesda/BSTEvent.h#L31l) although I agree it's superfluous

Reply:
We should prolly get rid of it since it's superfluous, code is marginally clearer without it